internal/exec/exec.go

```diff
@@ -0,0 +1,116 @@
+package exec
+
```

Comment:
Perhaps it is. I never considered it complicated enough to search for a third-party package that adds a little process management around the standard library's exec package. This is neatly encapsulated into a package so that it *can* act as a standalone package, if you want to publish one. I didn't feel like it adds enough features to warrant publishing as a standalone execution/process manager package.